lib/rails/generators/preview/templates/component_preview.rb.tt

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
```

Comment:
Do we need this line for previews to work correctly? If not, let's remove it:

```suggestion
```

Reply:
My bad, something I forgot to remove from the copy/paste operation. ;)